KDS-1865-fix-design-tokens-dep-in-kaio (#4198)
cultureamp/kaizen-design-system: packages/design-tokens/docs/pages/how-to-use-in-js.mdx

import { Meta } from "@storybook/blocks"
import { LinkTo } from "~storybook/components/LinkTo"

<Meta title="Systems/Tokens/Access tokens via JS" parameters={{ docsOnly: true }} />

# Access tokens via JSX

## 1. Set up a KaizenProvider

Ensure you have followed the steps in <LinkTo pageId="components-kaizen-provider-installation">KaizenProvider</LinkTo> to set up your application for using design tokens.

## 2. Use Tokens

### In JSX via CSS Vars

Because we use CSS vars, you can utilise them in your JSX as you would in CSS.

For example:

```jsx

const Component = () => (
  <div style={{
    backgroundColor: vars(--color-purple-500) // #844587
  }}>
    Custom Kaizen theme component
  </div>
)
```

### In JS functions

You can access the tokens directly:

```jsx
import { tokens } from "@kaizen/design-tokens/js"

const myAmazingFunction = () => {

>   const mainColor = data ? colors.purple[500]: colors.purple[300];

Reviewer: Should be this I think? Unless the import is the place to be changed.
```suggestion
  const mainColor = data ? tokens.colors.purple[500]: tokens.colors.purple[300];
```

> 

Reviewer: ```suggestion
```

import { Meta } from "@storybook/blocks"
import { LinkTo } from "~storybook/components/LinkTo"

<Meta title="Systems/Tokens/Access tokens via JS" parameters={{ docsOnly: true }} />

# Access tokens via JSX

## 1. Set up a KaizenProvider

> Ensure you have followed the steps in <LinkTo pageId="components-kaizen-provider-installation">KaizenProvider</LinkTo> to set up your application for using themes.

Reviewer: ```suggestion
Ensure you have followed the steps in <LinkTo pageId="components-kaizen-provider-installation">KaizenProvider</LinkTo> to set up your application for using design tokens.
```
